fullscreen/model/remove-last-manual.html

```diff
@@ -31,7 +31,7 @@
                 // When the asynchronous steps of "exit fullscreen" ran, only
                 // /first/ was in the top layer, and thus we exited fully.
                 assert_equals(document.fullscreenElement, null);
-                assert_equals(event.target, document);
+                assert_equals(event.target, first);
```

Comment:
This is also rather odd, because *first* is the fullscreen element when the async steps run, that's where the event will be fired, even though it's only the fullscreen element for a brief period.